### DISCUSSION TITLE
Error Inviting User

### DISCUSSION THREAD

**Discussion by tailorck** [Budibase/budibase#6050] [Help]
I get this error when I try to create a new user through the web app. Has anyone else experienced this?

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/7613002/170131054-16c02212-7597-491b-8f8a-02906914dd8f.png">

**Comment by melohagan** [comment]
Hi @tailorck 

Are there any error messages in the console and network tabs? (Fn+F12 on Chrome)
